libs/application/templates/parental-leave/src/lib/ParentalLeaveTemplate.ts

```diff
@@ -736,13 +752,37 @@ const ParentalLeaveTemplate: ApplicationTemplate<
           ],
           [DefaultEvents.ABORT]: [
             {
+              cond: (application) => goToState(application, States.APPROVED),
               target: States.APPROVED,
             },
+            {
+              cond: (application) =>
+                goToState(application, States.VINNUMALASTOFNUN_APPROVAL),
+              target: States.VINNUMALASTOFNUN_APPROVAL,
+            },
+            {
+              cond: (application) =>
+                goToState(application, States.EMPLOYER_EDITS_ACTION),
+              target: States.EMPLOYER_EDITS_ACTION,
+            },
+            {
+              cond: (application) =>
+                goToState(application, States.EMPLOYER_APPROVE_EDITS),
+              target: States.EMPLOYER_APPROVE_EDITS,
+            },
+            {
+              cond: (application) =>
+                goToState(
+                  application,
+                  States.EMPLOYER_WAITING_TO_ASSIGN_FOR_EDITS,
+                ),
+              target: States.EMPLOYER_WAITING_TO_ASSIGN_FOR_EDITS,
+            },
```

Comment:
You should only go to 'vinnumalastofnunApproval', 'vinnumalastofnunApprovalEdits' or 'approved' states when you abort changes